php/class-wp-cli.php

```diff
@@ -62,7 +62,12 @@ public static function get_configurator() {
 		static $configurator;
 
 		if ( ! $configurator ) {
-			$configurator = new Configurator( WP_CLI_ROOT . '/php/config-spec.php' );
+			if ( defined( 'WP_CLI_CONFIG_SPEC_PATH' ) ) {
+				$config_spec = WP_CLI_CONFIG_SPEC_PATH;
+			} else {
+				$config_spec = WP_CLI_ROOT . '/php/config-spec.php';
+			}
+			$configurator = new Configurator( $config_spec );
```

Comment:
One thing I'm uncertain about: should some global parameters _always_ be defined (e.g. `--debug`, `--prompt`, `--quiet`)?